core/test/database/database_soci_proxy_tests.cpp

```diff
@@ -80,6 +80,7 @@ class SociProxyBaseTest : public ::testing::Test {
         _backend = std::make_shared<ProxyBackend>(engine);
         _backend->enableQueryLogging(true);
         auto date = DateUtils::toJSON(std::chrono::system_clock::now());
+        std::remove(date.begin(), date.end(), ':');
```

Comment:
Why is that needed?

Reply:
":" can't be part of file name on windows, so the test fails.